charms/build-and-release-etcd.sh

```diff
@@ -45,6 +48,7 @@ touch report.xml
 
 if [ ${RELEASE} = true ]; then
   ETCD_CHARM=$(/usr/bin/charm push $JUJU_REPOSITORY/builds/etcd cs:~containers/etcd | head -n 1 | awk '{print $2}')
+  charm set ${CHARM} commit=${COMMIT_HASH}
```

Comment:
This one need to be `ETCD_CHARM`, not `CHARM`

Or better yet, feel free to rename `ETCD_CHARM` to `CHARM` elsewhere in this file, instead, to be consistent with the other build scripts.